lib/TPP/CombineXsmmPass.cpp

```diff
@@ -0,0 +1,194 @@
+//===CombinePass.cpp -------------------------------------------*----C++-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM
+// Exceptions. / See https://llvm.org/LICENSE.txt for license information. /
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+
+#include "TPP/BuilderUtils.h"
+#include "TPP/Dialect/Xsmm/XsmmOps.h"
+#include "TPP/Dialect/Xsmm/XsmmUtils.h"
+#include "TPP/Passes.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Dialect/SCF/IR/SCF.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+
+using namespace mlir;
+using namespace mlir::tpp;
+
+#define GEN_PASS_CLASSES
+#include "TPP/Dialect/Xsmm/XsmmUtils.h"
+#include "TPP/Passes.h.inc"
+#include "TPP/VNNIUtils.h"
+#include <iostream>
+namespace {
+
+static FailureOr<DenseI64ArrayAttr>
```

Comment:
is this copied from `ConvertToXsmm.cpp`?

Reply:
renamed